spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -51,6 +55,23 @@
       click_on "Show logs"
     end
 
+    context 'when an organsiation with no ips searches logs' do
+      before do
```

Comment:
I don't think this is needed.

If you go into this context, and override the `admin_user`, then the outer context's `before` does all the work.